go/vt/discovery/healthcheck.go

```diff
@@ -92,6 +92,14 @@ var (
 
 	// How much to sleep between each check.
 	waitAvailableTabletInterval = 100 * time.Millisecond
+
+	// HealthCheckCacheTemplate uses healthCheckTemplate with the `HealthCheck Tablet - Cache` title to create the
+	// HTML code required to render the cache of the HealthCheck.
+	HealthCheckCacheTemplate = fmt.Sprintf(healthCheckTemplate, "HealthCheck Tablet - Cache")
+
+	// HealthCheckHealthyTemplate uses healthCheckTemplate with the `HealthCheck Tablet - Healthy Tablets` title to
+	// create the HTML code required to render the list of healthy tablets from the HealthCheck.
+	HealthCheckHealthyTemplate = fmt.Sprintf(healthCheckTemplate, "HealthCheck Tablet - Healthy Tablets")
```

Comment:
Can possibly drop "Tablet", so that the headers show up as "HealthCheck - Cache" and "HealthCheck - Healthy Tablets"

Reply:
Fixed via f6d526e8da